grafana-plugin/src/containers/OutgoingWebhook2Form/OutgoingWebhook2Form.config.tsx

```diff
@@ -172,20 +172,20 @@ export const form: { name: string; fields: FormItem[] } = {
         rows: 2,
       },
     },
+    {
+      name: 'forward_all',
+      normalize: (value) => Boolean(value),
+      type: FormItemType.Switch,
+      description: "Forwards whole payload of the alert to the webhook's url as POST/PUT data",
+    },
     {
       name: 'data',
-      getDisabled: (form_data) => Boolean(form_data?.forward_whole_payload),
+      getDisabled: (data) => Boolean(data?.forward_all),
```

Comment:
Should this go into existing webhook too?

Reply:
Existing webhooks are already using the correct field name (`forward_whole_payload`).